docs/security/trust/trust_key_mng.md

```diff
@@ -64,6 +64,16 @@ Before backing them up, you should `tar` them into an archive:
 $ umask 077; tar -zcvf private_keys_backup.tar.gz ~/.docker/trust/private; umask 022
 ```
 
+## Hardware storage and signing
+
+Docker Content Trust can store and sign with root keys from a Yubikey 4.
```

Comment:
There are about four ideas in this and they are all smushed together.  Hard to parse. Try:

```markdown
Docker Content Trust can store and sign with root keys from a Yubikey 4. The
Yubikey is prioritized over keys stored in the filesystem. When you initialize a
new repository with content trust, the Engine looks for a root key locally. If a
key is not found and the Yubikey 4 exists, the Engine creates a root key in
Yubikey 4.
```